src/components/Layouts/Connected/Connected.layout.jsx

```diff
@@ -2,19 +2,81 @@ import './Connected.layout.scss';
 
 import React from 'react';
 import { Outlet } from 'react-router-dom';
-import Grid from '@mui/material/Grid';
 
+import { styled } from '@mui/material/styles';
+import { Box, Drawer, CssBaseline, Toolbar as MuiToolBar, IconButton } from '@mui/material';
+import { KeyboardArrowRight as OpenIcon } from '@mui/icons-material';
+
+import Header from '../../Sections/Header/Header';
 import Toolbar from '../../Sections/Toolbar/Toolbar';
 
-const ConnectedLayout = () => (
-  <Grid container>
-    <Grid item xs={0} sm={3} lg={2} display={{ xs: 'none', sm: 'block' }}>
-      <Toolbar />
-    </Grid>
-    <Grid item xs={12} sm={9} lg={10}>
-      <Outlet />
-    </Grid>
-  </Grid>
+const drawerWidth = 240;
+
+const Main = styled('main', { shouldForwardProp: (prop) => prop !== 'open' })(
+  ({ theme, open }) => ({
+    position: 'relative',
+    flexGrow: 1,
+    padding: theme.spacing(3),
+    transition: theme.transitions.create('margin', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+    marginLeft: `-${drawerWidth}px`,
+    ...(open && {
+      transition: theme.transitions.create('margin', {
+        easing: theme.transitions.easing.easeOut,
+        duration: theme.transitions.duration.enteringScreen,
+      }),
+      marginLeft: 0,
+    }),
+  })
 );
 
+const ConnectedLayout = () => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Box sx={{ display: 'flex' }}>
+      <CssBaseline />
+      <Header sx={{ zIndex: (theme) => theme.zIndex.drawer + 1 }} />
+      <Box sx={{ height: '100vh' }}>
+        <Drawer
+          variant="persistent"
+          anchor="left"
+          open={open}
+          sx={{
+            width: drawerWidth,
+            flexShrink: 0,
+            '& .MuiDrawer-paper': { width: drawerWidth, boxSizing: 'border-box' },
+          }}
+        >
+          <MuiToolBar />
+          <Box sx={{ overflow: 'auto', height: '100%' }}>
+            <Toolbar />
+          </Box>
+        </Drawer>
+      </Box>
+      <Main open={open}>
+        <IconButton
+          aria-label="Open menu"
+          onClick={() => setOpen(!open)}
+          size="large"
+          sx={{
+            position: 'absolute',
+            padding: '4px',
+            top: 0,
+            left: 0,
+            transform: open && 'rotate(180deg)',
+            borderRadius: 0,
+            opacity: 100,
+          }}
+        >
+          <OpenIcon />
```

Comment:
Suggest style for a more visible icon

<img width="55" alt="image" src="https://user-images.githubusercontent.com/54967480/207441582-681bccf6-0e95-4512-aa52-47c83eea445f.png">

<img width="301" alt="image" src="https://user-images.githubusercontent.com/54967480/207441673-282ccdac-2f7d-408a-b5b6-f26317659935.png">


```suggestion
            border: 'solid rgba(0, 0, 0, 0.12)',
            borderWidth: '1px 1px 1px 0',
            borderRadius: '0 4px 4px 0',
            position: 'absolute',
            padding: '4px',
            top: 3,
            left: 0,
            opacity: 100,
          }}
        >
          <OpenIcon sx={{ transform: open && 'rotate(180deg)' }} />
```